feat: UTC-302: As a role having the organization.invite permission I want to invite external users to a specific project directly

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/18140455140)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/624
<!-- Follow Merge -->

mcanu committed:
Changed invite permissions

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18019997791

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18020241340

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-302'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18020241340

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18020340072

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18030777805

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-302'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18030777805

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18041295848

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18099435142

mcanu committed:
Merge branch 'develop' into 'fb-utc-302'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18099435142

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18099582664

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18099860187

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18100051531

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18100226407

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18101436309

farioas commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18139904628)
> Successfully merged:  14 files changed, 125 insertions(+), 100 deletions(-)

farioas committed:
Merge branch 'develop' into 'fb-utc-302'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18139904628

farioas commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/18140455140)

farioas committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/18140455140
